Engine/game/savegame.h

```diff
@@ -52,7 +52,8 @@ enum SavegameVersion
     kSvgVersion_Cmp_64bit = 10,
     kSvgVersion_350_final = 11,
     kSvgVersion_350_final2= 12,
-    kSvgVersion_Current   = kSvgVersion_350_final2,
+    kSvgVersion_399       = 13,
```

Comment:
Please give new save format a much higher number in case there will be change in 3.* branch